scripts/generate_crds_manifests.sh

```diff
@@ -116,6 +116,43 @@ spec:
       name: quay.io/openshift/origin-kube-rbac-proxy:latest
 EOF
 
+cat << EOF > manifests/0000_50_olm_06-psm-operator.networkpolicy.yaml
```

Comment:
Why do we both have a version of this file and overwrite it in the script? If the script overwrite is needed for some reason, maybe a comment is in order to explain?

Reply:
the corresponding files in manifests is generated here

Reply:
I've added a header comment to the script